utils/compression.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+from __future__ import with_statement
+
+import logging
+import os
+import subprocess
+from string import Template
+
+logger = logging.getLogger(__name__)
+
+
+class Compressor(object):
+    """
+    Compress files.
+    """
+
+    def __init__(self, input=None, output=None, debug=False):
```

Comment:
Not sure what's the use case for compressor when `input` can have `None` as the default value? 
In any case `None` should be to handled  as a special case and raise `RuntimeError`